ja_jp/settings-writeEmail.adoc

```diff
@@ -0,0 +1,52 @@
+=== メッセージの作成
+==== 送信を取り消す
+*送信を取り消す* の機能使うと、送信ボタンを押した直後のメールについて、その送信リクエストを取り消すことができます。
+この機能を有効にすると、メールを送信するとすぐに *取り消す* リンクが表示されます。
+この通知は数秒間だけ表示されます。
+通知が消えると、送信の取り消しはできなくなります。
+
+この通知が画面に表示されている間は、メールは *下書き* フォルダに残っています。
+*取り消し* をクリックすると、送信がキャンセルされたことを知らせる通知が表示され、メールは *下書き* フォルダに残ります。
+
+. image:graphics/cog.svg[cog icon, width=20] -> *設定* を選択します。
+. *メッセージの作成* から、 *送信を取り消す* の *有効* チェックボックスにチェックを追加して、この機能を有効にします。
+. ページ下部のbtn:[保存]をクリックします。
+
+==== 開封確認を要求
+メールを送信する際に、受信先からメールの開封確認を常に要求することが可能です。
+
+. image:graphics/cog.svg[cog icon, width=20] -> *設定* を選択します。
+. *メッセージの作成* から、 *開封確認を要求* の *常時* チェックボックスにチェックを追加して、この機能を有効にします。
+. ページ下部のbtn:[保存]をクリックします。
+
+NOTE: メール作成中に受信者に開封確認を要求する方法については、<<mail-compose.adoc#_開封確認を要求>>をご覧ください。

```

Comment:
Check the link works after fixing `mail-compose.adoc`

Reply:
fixed

Reply:
fixed